Readme.md

```diff
@@ -14,49 +14,38 @@ interactions without compromising either performance or ease of use.
 - [API Reference &rarr;](https://deno.land/x/capi/mod.ts)<br />A generated API
   reference, based on type signatures and TSDocs.
 
-## At a Glance
+## Installation
 
-Run the Capi development server.
+We can substitute the URL's target (in this case the WebSocket URL of a proxy).
```

Comment:
What does this mean?

Reply:
Meaning that it can be swapped out with another URL / target